src/helpers/get-pull-requests.ts

```diff
@@ -83,9 +87,31 @@ export const getPullRequests = ({
         ),
         url: pr.node.url,
         repoUrl: pr.node.repository.url,
-        title: pr.node.title
+        title: pr.node.title,
+        createdAt: pr.node.createdAt
     }))
 
+    openPRs.sort(
```

Comment:
Looks good to me.

Just a thought, wouldn't it be better to chain the sorting implementation to each individual variable declaration, rather than having it all here?

Also, is the sorting supposed to show the oldest result first against the most recent result first?

Reply:
If I understand correctly, you mean we chain the sort to the filter? we can chain it such that we do the sort after the filter. But does it make any difference?
> Also, is the sorting supposed to show the oldest result first against the most recent result first?

No, its recent to oldest

Reply:
No difference technically but it aids the separation of concerns.